specs/deneb/fork-choice.md

```diff
@@ -93,6 +95,9 @@ def on_block(store: Store, signed_block: SignedBeaconBlock) -> None:
     # If not, this block MAY be queued and subsequently considered when blob data becomes available
     assert is_data_available(hash_tree_root(block), block.body.blob_kzg_commitments)
 
+    # *Note*: Extraneous or invalid Blobs (in addition to the expected/referenced valid blobs)
+    # received on the p2p network MUST NOT invalidate a block that is otherwise valid and available
+
```

Comment:
nitpicking: the comments are easy to ignore. It would be better to group this note with `is_data_available` tighter.

```python
    # [New in Deneb:EIP4844]
    # Check if blob data is available
    # If not, this block MAY be queued and subsequently considered when blob data becomes available
    # *Note*: Extraneous or invalid Blobs (in addition to the expected/referenced valid blobs)
    # received on the p2p network MUST NOT invalidate a block that is otherwise valid and available
    assert is_data_available(hash_tree_root(block), block.body.blob_kzg_commitments)
```

Reply:
Agree with this one